CMakeLists.txt

```diff
@@ -30,6 +30,8 @@ add_subdirectory(ASO_CBCT)
 add_subdirectory(AREG_IOS)
 add_subdirectory(AREG_CBCT)
 add_subdirectory(AREG)
+add_subdirectory(AutoMatrix)
+add_subdirectory(AutoMatrix/Matrix_CLI)
```

Comment:
This works, but I recommend you avoid skipping directories like this in `add_subdirectory` commands; it makes tracing CMake calls more difficult.

I'd suggest moving `Matrix_CLI` to the top-level, so that `AutoMatrix` and `Matrix_CLI` directories are siblings.

Alternatively, if you really prefer the current directory structure for organizational purposes, you should remove this line and instead add `add_subdirectory(Matrix_CLI)` to `AutoMatrix/CMakeLists.txt`.